### PR TITLE
fix(vue3): use value from v-model (instead of event.target.value)

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Ref #12366
  * To test - create a new conversation with description and passport via dialog window

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖥️ Tested with Desktop client or should not be risky for it

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
 			<router-view />
 		</NcAppContent>
 		<RightSidebar :is-in-call="isInCall" />
-		<MediaSettings :recording-consent-given.sync="recordingConsentGiven" />
+		<MediaSettings v-model.recording-consent-given="recordingConsentGiven" />
```
Comment (ShGKme) [inline src/App.vue:16]: Incorrect syntax

File: src/components/BreakoutRoomsEditor/SelectableParticipant.vue [hunk 2]
```diff
@@ -44,21 +44,21 @@ export default {
 			required: true,
 		},
 
-		checked: {
+		modelValue: {
```
Comment (ShGKme) [inline src/components/BreakoutRoomsEditor/SelectableParticipant.vue:47]: It is not a part of a breaking change 👀

File: src/components/TopBar/TopBar.vue [hunk 3]
```diff
@@ -4,7 +4,7 @@
 -->
 
 <template>
-	<div class="top-bar" :style="topBarStyle" :data-theme-dark="isInCall">
+	<div class="top-bar" :style="topBarStyle" :data-theme-dark="isInCall ? true : undefined">
```
Comment (ShGKme) [inline src/components/TopBar/TopBar.vue:7]: And this is far from `.sync` modifier changes 👀

Comment (Antreesy) [reply src/components/TopBar/TopBar.vue:7]: :shrug:

**Comment by ShGKme** [comment]
Converting `:prop.sync` syntax to `v-model:prop` is covered automatically by ESLint in https://github.com/nextcloud/spreed/pull/12378

So we can use this PR to fix other stuff like theming.

**Comment by Antreesy** [comment]
Rebased, only fixes are kept

**Review by ShGKme — APPROVED** [review]

**Pull request merged**